pkg/kcp/provider/aws/mock/vpcPeeringStore.go

```diff
@@ -2,40 +2,51 @@ package mock
 
 import (
 	"context"
-	"fmt"
 	ec2types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/elliotchance/pie/v2"
 	"k8s.io/utils/pointer"
 	"sync"
 )
 
 type VpcPeeringConfig interface {
-	AddVpcPeering()
+	AddVpcPeering(PeeringConnectionId, vpcId, remoteVpcId, remoteRegion, remoteAccountId string) *ec2types.VpcPeeringConnection
```

Comment:
why config and not a regular AWS API call? "config" are used to cover cases that are not supported by AWS API, but create peering sounds like part of the API